src/parse.rs

```diff
@@ -38,6 +41,10 @@ pub struct Bms {
     pub header: Header,
     /// The objects in the score.
     pub notes: Notes,
+    /// Lines that starts with #COMMENT.
+    pub comments: Vec<String>,
```

Comment:
`#COMMENT` lines are already stored in [`Header::comment`](https://github.com/MikuroXina/bms-rs/blob/4b07f2dae42676c7625de12cf84965a7b44ef1db/src/parse/header.rs#L53), so it is duplicated.